### PR TITLE
Pandas

### PR DESCRIPTION
**Pull request by jdpigeon**
Contains intro lesson to Pandas DataFrames in Python I presented in Fall of 2015

### CODE REVIEW

File: lessons/pandas/lesson.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+visible: true
+---
+<!-- change visible to true if you want it on the site -->
+
+# Intro to Pandas DataFrames
+
+ - **Authors**: Dano Morrison
+ - **Research field**: Neuroscience
+ - **Lesson topic**: Usage of Pandas DataFrames for data analysis
+
```
Comment (lwjohnst86) [inline lessons/pandas/lesson.md]: Could you include this below the `Lesson topic`:

```
- **Lesson content URL**: <https://github.com/UofTCoders/studyGroup/tree/gh-pages/lessons/pandas>
```

File: lessons/pandas/lesson.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+Pandas.GroupBy is a great function that allows you to process your data in many different ways without having to get fancy or write any loops.
+
+
+Essentially, it carries out three different steps:
+
+**Splitting** the data into different groups [eg. treatment condition]
+
+**Applying** some function to your data [eg. mean]
+
+**Combining** the results back into another DataFrame
```
Comment (lwjohnst86) [inline lessons/pandas/lesson.md]: You could convert this into a list via `-`

File: lessons/pandas/lesson.md [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+Pandas.GroupBy is a great function that allows you to process your data in many different ways without having to get fancy or write any loops.
+
+
+Essentially, it carries out three different steps:
+
+**-Splitting** the data into different groups [eg. treatment condition]
+
+**-Applying** some function to your data [eg. mean]
+
+**-Combining** the results back into another DataFrame
```
Comment (lwjohnst86) [inline lessons/pandas/lesson.md]: Oops, the `-` should be before the `**` and with a space too :) Like so `- **Word**`

Also, please remove the empty lines between, so its like:

```

For the website aesthetics :)
- **Word**
- **Word**
- **Word**

```

File: lessons/pandas/lesson.md (hunk 2 above)
Comment (jdpigeon) [inline lessons/pandas/lesson.md]: Hey Luke,

I made the changes. Do I send another pull request?

File: lessons/pandas/lesson.md [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+~~~
+pd.concat([movies,movies])
+~~~
+
+# The Amazing GroupBy
+
+Pandas.GroupBy is a great function that allows you to process your data in many different ways without having to get fancy or write any loops.
+
+Essentially, it carries out three different steps:
+- *Splitting*: the data into different groups [eg. treatment condition].
```
Comment (lwjohnst86) [inline lessons/pandas/lesson.md]: You almost got it! Sorry I didn't explain this part. List items can be tricky. They need an empty line before and after the list. So it should be:

```
Essentially, it carries out three different steps:

- *Splitting*: the data into different groups [eg. treatment condition].
```

Otherwise, no need to do another pull request. Thanks alot!!

Comment (jdpigeon) [inline lessons/pandas/lesson.md]: Think I got it!

**Comment by lwjohnst86** [comment]
Excellent, thanks!
